.travis.yml

```diff
@@ -17,6 +17,11 @@ addons:
     update: false
 
 script:
-- set -o pipefail && xcodebuild test -enableCodeCoverage YES -workspace Example/TABTestKit.xcworkspace -scheme TABTestKit_ExampleUITests -sdk $IPHONE_SIMULATOR_SDK -destination "OS=$OS_VERSION,name=$IPHONE_NAME" | xcpretty
-- carthage build --archive
-- pod lib lint
+  # Ensure SPM builds. SPM is only supported for Xcode 11 and newer, so we'll ignore this step if trying to build for iOS 12 (Xcode 10)
+  - if [ $OS_VERSION != '12.4' ]; then set -o pipefail && xcodebuild -workspace package.xcworkspace -scheme TABTestKit -sdk $IPHONE_SIMULATOR_SDK | xcpretty; fi
+  # Ensure Carthage builds
+  - carthage build --archive
+  # Ensure Cocoapods builds
+  - pod lib lint --verbose
+  # Ensure tests pass
+  - set -o pipefail && xcodebuild test -enableCodeCoverage YES -workspace Example/TABTestKit.xcworkspace -scheme TABTestKit-Example -sdk $IPHONE_SIMULATOR_SDK -destination "OS=$OS_VERSION,name=$IPHONE_NAME" | xcpretty
```

Comment:
How come you've moved the tests last?

Reply:
Because the tests take the longest to run, I was hoping we could get the whole thing to fail if it doesn't even build. There's no point even running the tests if it doesn't build for SPM/Carthage/Cocoapods right? Unfortunately it keeps running anyway

Reply:
Yeah fair enough, good point. There's a `fast_finish: true` option which might work

Reply:
I'll try adding that

Reply:
Added :)